app/src/main/res/layout/profile_reset_pin_activity.xml

```diff
@@ -62,14 +62,13 @@
 
           <TextView
             android:id="@+id/profile_reset_pin_main"
+            style="@style/Body"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
```

Comment:
layout_width and layout_height can be removed from this and other files as it is `wrap_content`


Ditto for all other files.

Reply:
Done.

Reply:
It seems this was wrong per the latest changes (& was reverted). @rt4914 can you confirm whether this comment is still relevant per the latest changes?

Reply:
This comment can be closed now. Thanks.